### DISCUSSION TITLE
en/latest/examples/workflows-examples/

### DISCUSSION THREAD

**Discussion by giscus** [argoproj-labs/hera#1088] [Website Discussions]
# en/latest/examples/workflows-examples/

https://hera.readthedocs.io/en/latest/examples/workflows-examples/

**Comment by jiantang99** [comment]
Hi, do you have any examples where a workflow template calls other workflow templates?

**Reply by elliotgunton** [reply]
See reply here https://github.com/argoproj-labs/hera/discussions/1089#discussioncomment-9660534
